app/services/user/invite.rb

```diff
@@ -9,6 +9,9 @@ def initialize(user:, organisation:)
   attr_reader :user, :organisation, :service
 
   def call
+    return unless user.service == :claims ||
+      Flipflop.enabled?(:user_onboarding_emails)
+
```

Comment:
Can this be more service specific? It seems sensible that the claims service may also want to disable onboarding emails, therefore, we want a similar flag which can be configured distinctly.

`placements_user_onboarding_emails` maybe?

```suggestion
    return if user.service == :placements && !Flipflop.enabled?(:placements_user_onboarding_emails)

```

-----

I also wonder if inverting the flag makes things _"nicer"_?

```suggestion
    return if user.service == :placements && Flipflop.enabled?(:placements_disable_user_onboarding_emails)

```

Reply:
I disagree with naming the flag `placements_disable_user_onboarding_emails`, otherwise this is resolved in [1191d14](https://github.com/DFE-Digital/itt-mentor-services/pull/523/commits/1191d1483859f3704dc435d07f9609e16164f3a8)